src/llmq/instantsend.cpp

```diff
@@ -1022,7 +1016,6 @@ std::unordered_set<uint256, StaticSaltedHasher> CInstantSendManager::ProcessPend
         if (it != recSigs.end()) {
             auto recSig = std::make_shared<CRecoveredSig>(std::move(it->second));
             if (!quorumSigningManager->HasRecoveredSigForId(llmqType, recSig->id)) {
-                recSig->UpdateHash();
```

Comment:
note: this was probably done to avoid hashing recsigs from bad nodes but I guess the impact of this is tiny comparing to the load caused by the bls verification itself, so should be ok to drop this creation/update split.